app/models/snapshot.rb

```diff
@@ -5,7 +5,12 @@ def self.get_from_source
     require 'open-uri'
     JSON.load(open("http://localhost:3001/acciones_mas_populares.json"))
   end
-  
+
+  def self.get_from_other_instance(dominio, snapshot_id)
+    require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.